File: templates/_composition/MVVMLight/Feature.UriSchemeExamplePage/ViewModels/ViewModelLocator_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class ViewModelLocator
    {
        public ViewModelLocator()
        {
            //^^
            //{[{
            Register<wts.ItemNameExampleViewModel, wts.ItemNameExamplePage>();
            //}]}
        }

        //{[{
        public wts.ItemNameExampleViewModel wts.ItemNameExampleViewModel => ServiceLocator.Current.GetInstance<wts.ItemNameExampleViewModel>();
        //}]}
    }
}

File: templates/Uwp/_composition/MVVMLight/Feature.UriSchemeExamplePageVB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Public Sub New()
            '^^
            '{[{
            Register(Of wts.ItemNameExampleViewModel, wts.ItemNameExamplePage)()
            '}]}
        End Sub

        '{[{

        Public ReadOnly Property wts.ItemNameExampleViewModel As wts.ItemNameExampleViewModel
            Get
                Return ServiceLocator.Current.GetInstance(Of wts.ItemNameExampleViewModel)
            End Get
        End Property
        '}]}
    End Class
End Namespace
